services/greasyfork/greasyfork-license.service.js

```diff
@@ -0,0 +1,42 @@
+import { renderLicenseBadge } from '../licenses.js'
+import { InvalidResponse } from '../index.js'
+import BaseGreasyForkService from './greasyfork-base.js'
+
+class GreasyForkLicense extends BaseGreasyForkService {
+  static category = 'license'
+
+  static route = {
+    base: 'greasyfork/l',
+    pattern: ':scriptId',
+  }
+
+  static examples = [
+    {
+      title: 'Greasy Fork',
+      namedParams: { scriptId: '407466' },
+      staticPreview: this.render({ licenses: ['MIT'] }),
+    },
+  ]
+
+  static defaultBadgeData = { label: 'license' }
+
+  static render({ licenses }) {
+    return renderLicenseBadge({ licenses })
+  }
```

Comment:
In cases where we're just utilizing a common helper like this there's no need to have a separate class-level render function, and it's fine to just call the helper directly instead at the call sites where `render` is being used

Reply:
I seem to get the error `render() function not implemented for GreasyForkInstalls` when removing `render`, do you know how to get around that?